README-Windows.md

```diff
@@ -0,0 +1,108 @@
+Fungible Token (FT)
+===================
+
+Example implementation of a [Fungible Token] contract which uses [near-contract-standards] and [simulation] tests. This is a contract-only example.
+
+**Note**: this README is specific to Windows and this example. For development on OS X or Linux, please see [README.md](README.md).
+
+  [Fungible Token]: https://nomicon.io/Standards/Tokens/FungibleTokenCore.html
+  [near-contract-standards]: https://github.com/near/near-sdk-rs/tree/master/near-contract-standards
+  [simulation]: https://github.com/near/near-sdk-rs/tree/master/near-sdk-sim
+
+Prerequisites
+=============
+
+If you're using Gitpod, you can skip this step.
+
+1. Make sure Rust is installed per the prerequisites in [`near-sdk-rs`](https://github.com/near/near-sdk-rs#pre-requisites)
+2. Ensure `near-cli` is installed by running `near --version`. If not installed, install with: `npm install --global near-cli`
+
+## Building
+
+To build run:
+```bash
+./build.sh
```

Comment:
I think we'll probably need to change this to be the cargo command:
    
    cd ft
    cargo build --target wasm32-unknown-unknown --release
    
Let's get rid of `flags.sh` and everything referencing it. The way to make this cross-platform is to add this `.cargo` directory inside the `ft` folder. Please copy it from here:

https://github.com/near-examples/rust-fungible-token/tree/master/.cargo

Reply:
`build.bat` added, because we need to copy `.wasm` files after the build